backend/yarn.lock

```diff
@@ -272,14 +350,73 @@ debug@2.6.9:
   version "2.6.9"
   resolved "https://registry.yarnpkg.com/debug/-/debug-2.6.9.tgz#5d128515df134ff327e90a4c93f4e077a536341f"
   integrity sha512-bC7ElrdJaJnPbAP+1EotYvqZsb3ecl5wi6Bfi6BJTUcNowp6cvspg0jXznRTKDjm/E7AdgFBVeAPVMNcKGsHMA==
-  dependencies:
-    ms "2.0.0"
-
-debug@3.1.0:
-  version "3.1.0"
-  resolved "https://registry.yarnpkg.com/debug/-/debug-3.1.0.tgz#5bb5a0672628b64149566ba16819e61518c67261"
-  integrity sha512-OX8XqP7/1a9cqkxYw2yXss15f26NKWBpDXQd0/uK/KPqdQhxbPa994hnzjcE2VqQpDslf55723cKPUOGSmMY3g==
-  dependencies:
+>>>>>>> 9f3f9283bc5dfadd4b3254ea9c1cc91181621871
```

Comment:
Here as well you have merge conflict messages unhandled
Kindly do not push this yarn.lock file in your commits

Reply:
Can you guide me how to remove it? Should I reset this commit using soft and then include every file except this yarn.lock file?

Reply:
Yes you can do that. When you add extra dependencies, just make sure they are added in the package.json file.

Reply:
Please check now.